### PR TITLE
Add basic Storybook setup

### PR DESCRIPTION
**Pull request by jbmoelker**
@kasszz @Ex-iT maybe one of you wants to pick this up. The basic setup is working. Though I'm not sure Storybook is entirely ready for Vue projects yet. You can open Storybook using:
`npm run dev:guide`.

### CODE REVIEW

**Comment by jbmoelker** [comment]
@kasszz @Ex-iT I think this one is ready to be reviewed / merged.

File: config/storybook/config.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { configure } from '@storybook/vue'
+import '../../src/core'
```
Comment (kasszz) [inline config/storybook/config.js:2]: Folder paths should end with a `/`, right ?

File: config/storybook/config.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+const req = require.context(`../../src/components`, true, /\.stories\.js$/)
```
Comment (kasszz) [inline config/storybook/config.js:4]: using backticks without needing variables in that string.

File: src/components/browser-viewport/browser-viewport.stories.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+})
+
+storiesOf('browser-viewport', module)
+  .addDecorator(withKnobs)
+  .add('Input URL', () => {
+    const url = text('URL', 'https://www.voorhoede.nl')
+    return createStory({ url })
+  })
+  .add('Using iframe', () => createStory({ url: 'https://www.voorhoede.nl' }))
+  .add('Using image ', () => createStory({ url: 'https://www.tradus.com' }))
```
Comment (kasszz) [inline src/components/browser-viewport/browser-viewport.stories.js:21]: Extra space?

File: src/components/editor/editor.stories.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+// import { storiesOf } from '@storybook/vue'
```
Comment (kasszz) [inline src/components/editor/editor.stories.js]: Whole file is one big comment?

**Review by kasszz — APPROVED** [review]

**Pull request merged**